.travis.mk

```diff
@@ -374,10 +380,17 @@ test_osx_no_deps: build_osx
 	${INIT_TEST_ENV_OSX}; \
 	cd test && ./test-run.py --vardir ${VARDIR} --force $(TEST_RUN_EXTRA_PARAMS)
 
+test_osx_arm64_only:
+	make PUC-Rio-Lua-5.1-tests
```

Comment:
I've merged several fixes in LuaJIT [recently](https://github.com/tarantool/tarantool/commit/bef585a), so let's also add `lua-Harness-tests` and `tarantool-tests` targets.

Reply:
Done.